src/lib/components/layouts/Error.svelte

```diff
@@ -24,6 +28,11 @@
     <slot name="message" />
   </Error>
   <slot />
+  {#if $page.status === 404 && !me}
```

Comment:
Message isn't showing up in the preview environment. I think $page.status isn't set.